config/prow/cluster/starter/starter-azure.yaml

```diff
@@ -93,11 +93,18 @@ data:
             [Full PR test history](https://prow.<< your-domain.com >>/pr-history?org={{.Spec.Refs.Org}}&repo={{.Spec.Refs.Repo}}&pr={{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}).
             [Your PR dashboard](https://prow.<< your-domain.com >>/pr?query=is:pr+state:open+author:{{with
             index .Spec.Refs.Pulls 0}}{{.Author}}{{end}}).
-      default_decoration_configs:
-        "*":
+      default_decoration_config_entries:
+        - config:
```

Comment:
Are these lines necessary? If so, can you explain why in the commit message description?

Reply:
Based on the comments on https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L669 it seems like `default_decoration_config_entries` is the new better way to do it so I also upgraded this example config to use that format.

Reply:
I updated the PR description with this explanation as well.  If you're not comfortable with the drive-by change I could easily revert it.  It's definitely not necessary.

Reply:
> Based on the comments on https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L669 it seems like `default_decoration_config_entries` is the new better way to do it so I also upgraded this example config to use that format.

Makes sense. Thanks.